spec/fixtures/w3c_distributed_tracing.json

```diff
@@ -1,92 +0,0 @@
-//
```

Comment:
Was this file removed from the specs?

Reply:
No, I just misread the diff, fixing it 👍